src/common/BaseAnnotator.ts

```diff
@@ -71,12 +74,26 @@ export default class BaseAnnotator {
         this.removeListener(LegacyEvent.SCALE, this.handleScale);
         this.removeListener(Event.ACTIVE_SET, this.setActiveAnnotationId);
         this.removeListener(Event.VISIBLE_SET, this.setVisibility);
+        document.removeEventListener('click', this.handleClick);
     }
 
     getElement(selector: HTMLElement | string): HTMLElement | null {
         return typeof selector === 'string' ? document.querySelector(selector) : selector;
     }
 
+    handleClick({ target }: MouseEvent): void {
+        const targetElement = target as HTMLElement;
+
+        if (
+            // If click target is not a part of the annotated element
+            (this.rootEl && !this.rootEl.contains(targetElement)) ||
```

Comment:
If this is something specific to the Preview container element, should it live in the Preview SDK? I've been trying to avoid UI-related stuff in the annotators and managers.